labelled_nfs/client.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+#
+# Environment variables used:
+#  - SERVER: hostname or IP-address of the NFS-server
+#  - EXPORT: NFS-export to test (should start with "/")
+
+# if any command fails, the script should exit
+set -e
+
+# enable some more output
+set -x
+
+[ -n "${SERVER}" ]
+[ -n "${EXPORT}" ]
+
+# install build and runtime dependencies
+yum -y install git gcc nfs-utils time
+
+# v4.2 mount
+mkdir -p /mnt/nfsv42
+mount -t nfs -o vers=4.2 ${SERVER}:${EXPORT} /mnt/nfsv42
+
+#create dir and file
+mkdir /mnt/nfsv43/dir && touch /mnt/nfs42/dir/file.txt
```

Comment:
This says nfs43.  Presumably should be nfs42?

Reply:
> This says nfs43. Presumably should be nfs42?

Thank you for the comment Dan, corrected the change.